io.catenax.iot_sensor_data/1.0.0/IoTSensorData.ttl

```diff
@@ -0,0 +1,275 @@
+:BatteryLevelCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Battery Level Characteristic"@en;
+    bamm:description "Characteristic to describe the property battery level"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:percent.
+:HumidityCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Humidity Characteristic"@en;
+    bamm:description "Characteristic that describes the property humidity"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:percent.
+:FillingLevelCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Filling Level Characteristic"@en;
+    bamm:description "Characteristic that describes the property filling level"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:percent.
+:TemperatureCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Temperature Characteristic"@en;
+    bamm:description "Characteristic that describes the property temperature"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:degreeCelsius.
+:LightCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Light Characteristic"@en;
+    bamm:description "Characteristic that describes the property light "@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:lumen.
+:AccelerationCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Acceleration Characteristic"@en;
+    bamm:description "Characteristic that describes the property acceleration"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:metrePerSecondSquared.
+:WeightCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Weight Characteristic"@en;
+    bamm:description "Characteristic that describes the property weight"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:kilogram.
+:PressureCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Pressure Characteristic"@en;
+    bamm:description "Characteristic that describes the property pressure"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:pascal.
+:BinarySensorCharacteristic a bamm-c:Enumeration;
+    bamm:preferredName "Binary Sensor Characteristic"@en;
+    bamm:description "Characteristic that describes the property binary sensor\n"@en;
+    bamm:dataType xsd:integer;
+    bamm-c:values ("0"^^xsd:integer "1"^^xsd:integer).
+:SoundSensorCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Sound Sensor Characteristic"@en;
+    bamm:description "Characteristic that describes the property sound sensor"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:decibel.
+:TransmissionMethodCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Transmission Method Characteristic"@en;
+    bamm:description "Characteristic that describes the property transmission method"@en;
+    bamm:dataType xsd:string.
+:AnalogSensorCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Analog Sensor 1-10 Characteristic"@en;
+    bamm:description "Characteristic that describes the property analog sensor"@en;
```

Comment:
What is this sensor about? I do not fully understand what 1-10 means? Is it that it can be between 1 and 10?

Reply:
After discussion with the team, there was some misunderstanding here. This attribute should be modeled as a list. Analog Sensor 1-10 means  the data is collected  from 10 sensors. I have modeled it now in a list.